### PR TITLE
Merge inventory system and related fixes with the master branch

### LINKED ISSUE #15: Player class completed
This issue tracks the player class. And when it is completed. This includes holding items and also replacing equipment.

### PR DESCRIPTION
**Pull request by 133794m3r**
With these commits we're one step closer to GM.
Closes #15, #32, #3

### CODE REVIEW

**Comment by 133794m3r** [comment]
Waiting on review from @NathanielMullins .

**Review by NathanielMullins — APPROVED** [review]
everything looks good on this end

**Comment by 133794m3r** [comment]
Merging in 5..4..3..2..1..

**Pull request merged**
